libs/core/src/lib/select/select.component.html

```diff
@@ -44,7 +44,7 @@
         class="fd-select__control"
         role="listbox"
         #selectControl
-        [ngClass]="state ? 'is-' + state : ''"
+        [ngClass]="(state ? 'is-' + state : '') + (panelOpen ? 'fd-select--hide-outline' : '')"
```

Comment:
I think you need space here ' fd-select--hide-outline' otherwise will append to `is-state` 
You can probably use it this way:
[ngClass]="state ? 'is-' + state : ''"
[class.fd-select--hide-outline]="panelOpen"